tests/functional_tests/keras/keras_flops_functional_tf_2_6.yea

```diff
@@ -0,0 +1,18 @@
+id: 0.keras.flopsfunctional.tf26
```

Comment:
hmmm... do we care testing against earlier version of tf?

Reply:
prob not, it's been there (unchanged) since 2.0

Reply:
what we should care about is testing more recent tf versions lol ;0

Reply:
yeah... that's also a good point